fvm/fvm_test.go

```diff
@@ -1303,11 +1302,8 @@ func TestBlockContext_GetBlockInfo(t *testing.T) {
 		ledger := testutil.RootBootstrappedLedger(vm, ctx)
 		require.NoError(t, err)
 
-		assert.PanicsWithValue(t, interpreter.ExternalError{
-			Recovered: logPanic{},
-		}, func() {
-			_ = vm.Run(blockCtx, fvm.Transaction(tx, 0), ledger, programs.NewEmptyPrograms())
-		})
+		err = vm.Run(blockCtx, fvm.Transaction(tx, 0), ledger, programs.NewEmptyPrograms())
+		require.Error(t, err)
```

Comment:
I updated these two testcases since cadence now handles these cases gracefully (according to https://github.com/onflow/cadence/pull/1364/files) 

@turbolent @janezpodhostnik just checking if this is the expected behaviour

Reply:
The error will be classified as a failure error in `SplitErrorTypes` and will crash the node so I think this is ok.

Reply:
👍